core/federated/RTI/rti_common.c

```diff
@@ -18,6 +18,20 @@ static rti_common_t* rti_common = NULL;
 // Global variables defined in tag.c:
 extern instant_t start_time;
 
+static tag_t latest_earlier_tag(tag_t tag) {
+  if (tag.time == NEVER || tag.time == FOREVER) {
```

Comment:
This is a nit, but strictly speaking, I think these should be using `lf_tag_compare` to compare with `NEVER_TAG` and `FOREVER_TAG`.

Reply:
Resolved in [896e152](https://github.com/lf-lang/reactor-c/pull/349/commits/896e15293bd21958a124d7f73b02c455340f81d7).